x-pack/plugins/aiops/public/components/log_categorization/log_categorization_for_embeddable/embeddable_menu.tsx

```diff
@@ -0,0 +1,157 @@
+  setSelectedField: (field: DataViewField) => void;
+  minimumTimeRangeOption: MinimumTimeRangeOption;
+  setMinimumTimeRangeOption: (w: MinimumTimeRangeOption) => void;
+  categoryCount: number | undefined;
+  reload: () => void;
+}
+
+const minimumTimeRangeOptions = Object.keys(MINIMUM_TIME_RANGE).map((value) => ({
+  inputDisplay: value,
+  value: value as MinimumTimeRangeOption,
+}));
+
+export const EmbeddableMenu: FC<Props> = ({
+  randomSampler,
+  fields,
+  selectedField,
+  setSelectedField,
+  minimumTimeRangeOption,
+  setMinimumTimeRangeOption,
+  categoryCount,
+  reload,
+}) => {
+  const [showMenu, setShowMenu] = useState(false);
+  const togglePopover = () => setShowMenu(!showMenu);
+
+  const fieldOptions = useMemo(
+    () => fields.map((field) => ({ inputDisplay: field.name, value: field })),
+    [fields]
+  );
+
+  const button = (
+    <EuiToolTip
+      content={i18n.translate('xpack.aiops.logCategorization.embeddableMenu.tooltip', {
+        defaultMessage: 'Options',
+      })}
+    >
+      <EuiButtonIcon
+        data-test-subj="aiopsEmbeddableMenuOptionsButton"
+        size="s"
+        iconType="controlsHorizontal"
+        onClick={() => togglePopover()}
+        // @ts-ignore - subdued does work
+        color="subdued"
+        aria-label={i18n.translate('xpack.aiops.logCategorization.embeddableMenu.aria', {
+          defaultMessage: 'Pattern analysis options',
+        })}
+      />
+    </EuiToolTip>
+  );
+
+  return (
+    <EuiPopover
+      id={'embeddableMenu'}
+      button={button}
+      isOpen={showMenu}
+      closePopover={() => togglePopover()}
+      panelPaddingSize="s"
+      anchorPosition="downRight"
+    >
+      <EuiPanel color="transparent" paddingSize="s" css={{ maxWidth: '400px' }}>
+        <EuiTitle size="xxxs">
+          <h3>
+            <FormattedMessage
+              id="xpack.aiops.logCategorization.embeddableMenu.patternAnalysisSettingsTitle"
+              defaultMessage=" Pattern analysis settings"
+            />
+          </h3>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiFormRow
+          data-test-subj="aiopsEmbeddableMenuSelectedFieldFormRow"
+          label={i18n.translate(
+            'xpack.aiops.logCategorization.embeddableMenu.selectedFieldRowLabel',
+            {
+              defaultMessage: 'Selected field',
+            }
+          )}
+        >
+          <EuiSuperSelect
+            aria-label="Select a field"
+            options={fieldOptions}
+            valueOfSelected={selectedField ?? undefined}
+            onChange={setSelectedField}
+          />
+        </EuiFormRow>
+
+        <EuiHorizontalRule margin="m" />
+
+        <EuiFormRow
+          data-test-subj="aiopsRandomSamplerOptionsFormRow"
+          label={i18n.translate(
+            'xpack.aiops.logCategorization.embeddableMenu.minimumTimeRangeOptionsRowLabel',
+            {
+              defaultMessage: 'Minimum time range',
```

Comment:
This field could do with an info tooltip. Challenge will be how to simply explain what it does :)

Reply:
Added tooltip LGTM